src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactElasticWellbore/Example.rst

```diff
@@ -0,0 +1,159 @@
+
+--------------------------------------------------------------------	       
+Numerical Methods
+--------------------------------------------------------------------
+
+The ``stabilizationName`` that is required in the ``LagrangianContact`` solver is defined by: 
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedElasticWellbore_ImperfectInterfaces_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_NumericalMethods -->
+  :end-before: <!-- SPHINX_NumericalMethodsEnd -->
+
+--------------------------------------------------------------------	       
+Contact region and material
+--------------------------------------------------------------------
+
+The imperfect contact surfaces between casing, cement, and rock layers are defined as ``Fracture`` as shown below:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedElasticWellbore_ImperfectInterfaces_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_SurfaceElementRegion -->
+  :end-before: <!-- SPHINX_SurfaceElementRegion -->
+
+Here, the ``faceBlock`` name, ``faceElementSubRegion``, is needed to define ``Tasks`` for collecting displacement jumps across the contact surfaces. The ``defaultAperture`` defined in this block is the default hydraulic aperture that should not be confused with the mechanical aperture. For this purely mechanical problem, the default hydraulic aperture parameter is omitted. The fracture material given in the ``materialList`` is defined as follows:
+
+.. literalinclude:: ../../../../../../../inputFiles/wellbore/CasedElasticWellbore_ImperfectInterfaces_base.xml
+  :language: xml
+  :start-after: <!-- SPHINX_MaterialContact -->
+  :end-before: <!-- SPHINX_MaterialContactEnd -->
+
+For this purely mechanical problem, without fluid flow and shearing stress acting on the contact surface, all the parameters defined in this block are omitted.
+
+---------------------------------
+Results and benchmark
+---------------------------------
+
+The GEOSX results of displacement jump across the casing-cement and cement-rock interfaces are shown in the figure below: 
+
+.. _CasedElasticWellboreInterfacesDisplacementJump3DFig:
+.. figure:: displacementJump3D.png
+   :align: center
+   :width: 600
+   :figclass: align-center
+
+   Displacement jumps across the casing-cement and cement-rock interfaces
+
+We can observe an expected zero displacement jump at the cement-rock interface under a tension stress on the inner surface of the casing. This is because the applied stress does not cause any strain on the cement and rock layers after the debonding occurs at the casing-cement interface. The displacement jump at the casing-cement interface is homogeneous and varies over time because the tension stress on the inner surface of the casing varies with time as defined in the XML file. A perfect comparison of GEOSX results versus theoretical results is shown in the figure below:
```

Comment:
```suggestion
We can observe an expected zero displacement jump at the cement-rock interface under a tension stress on the inner surface of the casing. This is because the applied stress does not cause any strain on the cement and rock layers after the debonding occurs at the casing-cement interface. The displacement jump at the casing-cement interface is homogeneous and varies over time because the tension stress on the inner surface of the casing varies with time as defined in the XML file. A perfect agreement between GEOs results and theoretical results is shown in the figure below:
```

Reply:
done